ts/Extensions/DashboardPlugins/DataGridComponent.ts

```diff
@@ -153,6 +178,7 @@ class DataGridComponent extends Component<DataGridComponent.ChartComponentEvents
      *
      * */
 
+    /** @internal */
```

Comment:
Can you please add a block descriptions everywhere or change the rules for that folder?